apps/docs/content/guides/auth/social-login/auth-github.mdx

```diff
@@ -119,7 +119,7 @@ func signInWithGithub() async throws {
 </TabPanel>
 <TabPanel id="kotlin" label="Kotlin">
 
-When your user signs in, call [signInWith(Provider)](/docs/reference/kotlin/auth-signinwithoauth) with `Github` as the `Provider`:
+When your user signs in, call [signInWith(Provider)](/docs/reference/kotlin/auth-signinwithoauth) with `GitHub` as the `Provider`:
```

Comment:
Not sure about this change, `Github` is coming from the Kotlin library.

Reply:
@ivasilov Great catch! It seems that our Kotlin library does expect it [that way](https://github.com/supabase-community/supabase-kt/blob/f1c71bd098e45bba984c9cca6197a34e9d283860/Auth/src/commonMain/kotlin/io/github/jan/supabase/auth/providers/Providers.kt#L16-L20). I'll update this bit.